src/components/Calculator.js

```diff
@@ -56,7 +57,12 @@ const Calculator = () => {
           </tr>
         </tbody>
       </table>
+      <div className="quotesDiv">
+        <Quotes />
```

Comment:
As per the project requirements, the `Quotes` component should be imported into the main component which is `App.js`. Kindly follow-up on this requirement.